src/scripts/generate-hounds.ts

```diff
@@ -81,3 +85,19 @@ export async function getHoundRenderer(): Promise<CanvasRenderFunc> {
     });
   };
 }
+
+// converts a number decoded from the Genome quadlet into an index that can be used
+// to look up the `parts.<body-part>` array.
+const MaxQuadLetValue = decodeBase62Quadlet("ZZZZ");
+export function genomeNumberToImageIndex(base10Quadlet: number): number {
+  assert(base10Quadlet >= 0 && base10Quadlet <= MaxQuadLetValue);
+  for (let i = 0; i < MaxQuadLetValue; ++i) {
```

Comment:
Any particular reason the limit is MaxQuadLetValue? It shouldn't matter too much I guess but just confirming.

Reply:
If we have an argument greater than `MaxQuadLetValue`, then it cannot be mapped to an index in the range 1 - 5.
Currently we're dividing the range of numbers 0 - ZZZZ in 5 equal slices, then mapping `base10Quadlet` to an index by finding out which slice it belongs to. If it was `MaxQuadLetValue + 1` for example, the index would be 5, but we only have 5 valid feature images for each body part (i.e max index possible is 4)

Reply:
Although I did understand this, I was actually wondering why your loop checks whether i is less then MaxQuadletValue. It wouldn't make a difference but I felt that it would be better understood if we replaced it by the number of partitions. (i.e 5)

Reply:
Ah, I think you're right!
The range should go from 0 to 5 instead of 0 to ZZZZ.
Good catch.